src/components/Comments/ReplyCommentItem.js

```diff
@@ -38,6 +46,7 @@ const ReplyCommentItem = ({
   updateVoteParent,
   onLongPress
 }) => {
+  const refSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `refSheet` is declared but not initialized. It should be initialized with `null`.
```diff
-  const refSheet = React.useRef();
+  const refSheet = React.useRef(null);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->